region/maridia/inner-green.json

```diff
@@ -1211,6 +1211,64 @@
                     ]}
                   ],
                   "devNote": "Other weapons take too long to kill the enemy."
+                },
+                {
+                  "name": "Cross Room Jump with Damage Boost",
+                  "notable": false,
+                  "requires": [
+                    "canCrossRoomJumpIntoWater",
+                    "canPlayInSand",
+                    "canDamageBoost",
+                    {"enemyDamage": { "enemy": "Evir", "type": "contact", "hits": 1 }},
+                    {"adjacentRunway": {
+                      "fromNode": 1,
+                      "usedTiles": 7,
+                      "physics": ["air"]
+                    }}
+                  ],
+                  "note": [
+                    "Requires a runway of approximately 7 tiles in the adjacent room.",
```

Comment:
This feels like a frame perfect door jump into a frame perfect aim down.
An easier alternative would be to kill the 2nd evir with plasma or a super and jump at the end of the runway.

Reply:
And the jump is before room entry.